index.js

```diff
@@ -1 +1,15 @@
-export * from './dist/auro-flight';
+import { AuroFlight } from './src/auro-flight';
+
+/**
+ * Register Custom Element.
+ * @param {Object} name - Name to use for custom element.
+ * @returns {void}
+ */
+ const registerComponent = (name = 'custom-flight') => {
+  // alias definition
+  if (!customElements.get(name)) {
+    customElements.define(name, class extends AuroFlight {});
+  }
+}
+
+export { registerComponent }
```

Comment:
Please set your IDE to always add a line after.

Reply:
Don't forget to update the following file/line as part of this update

https://github.com/AlaskaAirlines/auro-flight/blob/fajarapri/node18update/.npmignore#L21